roles/ingress/defaults/main.yml

```diff
@@ -38,5 +38,9 @@ ingress_class_name: "{{ atmosphere_ingress_class_name }}"
 #               annotations to all Ingress resources.
 ingress_annotations: {}
 
+# Seperate cert manager annotations so ingress_annotations can be override.
+ingress_cert_annotations:
+  cert-manager.io/common-name: "{{ ingress_host }}"
```

Comment:
@ricolin is there a reason we can't keep the existing template and not introduce a new variable to handle this?

Reply:
depends on how we like the override to works here.
We can totally write a hard code dict in template like ansible.builtin.combine({cert-manager.io/common-name: "{{ ingress_host }}"}
or put this cert-manager stuff into `ingress_annotations`

Reply:
like this?
```
  annotations:
      {{ { cert-manager.io/common-name: {{ ingress_host }} } | combine(ingress_annotations, recursive=True) | combine(ingress_global_annotations, recursive=True) | to_nice_yaml | indent(4) }
```

Reply:
updated, please check :)

Reply:
Is there no way of keeping the annotation hard coded the way it is and then adding the extra annotation on top of it?

Reply:
Another way is to check keys before parse to nice yaml function so {} won't be generated.
Just updated again. Please check. Thanks